packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -188,6 +211,34 @@ export async function produceBlockBody<T extends BlockType>(
           if (payload.transactions.length === 0) {
             this.metrics?.blockPayload.emptyPayloads.inc({prepType});
           }
+
+          // Capella and later forks have withdrawals on their ExecutionPayload
+          if (fork === ForkName.capella || fork === ForkName.eip4844) {
+            // TODO EIP-4844 Remove this when the EC includes `withdrawals`
+            (blockBody as capella.BeaconBlockBody).executionPayload.withdrawals = [];
```

Comment:
this will break the withdrawals, so we should do this only in an unmerged PR

Reply:
removing from here, so that we can only keep this in the unmerged eip-4844 parent PR

Reply:
Sure, we should merge withdrawal logic here ASAP. Is it there already on unstable or no-where yet?

Reply:
merge withdrawal logic is already in the executin/http where withdrawals are bundeled on execution payload depending on fork

Reply:
we set withdrawal to attributes here: https://github.com/ChainSafe/lodestar/pull/4864/files#diff-c331f7b8e7121839c55f5f20f21d1efe1375b69c6fbbe5db12a4baf1ed4acf5dR350, and rest execution apis handle it appropriately